plugins/salesforce/komand_salesforce/util/api.py

```diff
@@ -368,11 +368,18 @@ def get_error(response: str) -> Tuple[str, str, Any]:
                     401,
                 )
             if error_desc:
-                return (
-                    f"Salesforce error: '{error_desc}'",
-                    PluginException.assistances[PluginException.Preset.UNKNOWN],
-                    "",
-                )
+                if "retry your request" in error_desc:
+                    return (
+                        f"Salesforce error: '{error_desc}'",
+                        PluginException.assistances[PluginException.Preset.UNAUTHORIZED],
+                        500,
+                    )
+                else:
+                    return (
```

Comment:
if we hit this else can we add a new logger to mention that we've hit an unhandled exception for getting an oauth token? it should let us update our internal alerts to only notify us of edge cases/responses we don't have specific error handling for